velox/functions/prestosql/Pi.h

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright (c) Facebook, Inc. and its affiliates.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#pragma once
+
+#include "velox/functions/Macros.h"
+
+namespace facebook::velox::functions {
+
+template <typename T>
+struct PiFunction {
+ FOLLY_ALWAYS_INLINE bool call(double& result) {
+   result = 3.14;
```

Comment:
Is this the value returned in Presto ? I believe value returned is to much higher precision (3.14159265358979323846).